helmfile/charts/networkpolicy-service/values.yaml

```diff
@@ -87,3 +87,18 @@ monitoring:
         - "0.0.0.0/0"
       ports:
         - 443
+certManager:
+  enabled: true
+# letsencrypt ip addresses
+  namespaces: []
```

Comment:
I don't think we need it in sc, so I vote for deleting it here.

Reply:
removed it !!

Reply:
Hmm still looks like it's here to me?

Reply:
yes

Reply:
I added it back, because, 

we want it to be installed in all namespaces by default. 

So , i exposed it and set it as an empty list by default.

Reply:
I agree that we don't need it in all namespaces. But I'm ok with adding it to the namespaces where we have ingresses via this template instead of adding the same thing in each different components netpols.

Reply:
Yes!! I correct my comment, we want it installed in all namespaces with ingress !!

Reply:
:+1: Then I'll remove this for opensearch